extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,147 @@
+
+## Modeling Absorption
+
+Absorption is usually defined as an amount of light at each frequency that is absorbed over a given distance through a medium (usually described by Beer’s Law). However, since this extension deals exclusively with very thin surfaces, we can treat absorption as a constant. In fact, rather than absorbed light, we can talk about its inverse: transmitted light. The `baseColor` of the material serves this purpose as it already defines how the light that penetrates the surface is colored by the material. In this model, the transmitted light will be modulated by this color as it passes through.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>The baseColor of the material is used to tint the light being transmitted.</em></figcaption>
+</figure>
+
+### Modeling Refraction
+
+Since the surface is considered to be infinitely thin, we will ignore macroscopic refraction caused by the orientation of the surface. However, microfacets on either side of the thin surface will cause light to be refracted in random directions, effectively blurring the transmitted light. This microfacet lobe is exactly the same as the specular lobe except sampled along the line of sight through the surface.
+
+
+## Realtime Implementation Notes
+
+Implementations of transmission are expected to vary, especially in real-time engines, depending on the capabilities of the runtime, but should endeavor to achieve plausibility. 
+
+
+### Refractive Blurring
+Some rasterizer implementations may opt to sample prefiltered IBL or dynamic light probes to simulate refractive blurring due to surface roughness. Others may choose to sample from a blurred version of the rendered scene behind a surface. Raytracers are expected to use more physically correct refraction with a BSDF supporting multi-scattering like the modified [Smith](https://eheitzresearch.wordpress.com/240-2/) model.
+
+<figure>
+  <img src="./figures/Roughness.png"/>
+<figcaption><em>Refraction due to surface roughness.</em></figcaption>
+</figure>
+
+**TODO** - *This is fine for raytracers or for rasterizers that just sample the existing IBL for refracted light. However, what about rasterizers that want to cheaply render transmission in screen-space (I think this is pretty common)? Typically, the refractive blurring is done by sampling mips or manually-blurred versions of the background scene. This can be eye-balled by an implementation but is there a slightly more mathematically-rigorous version that we could recommend?*
+
+### Blending
+
+The glTF `blendMode` is used for alpha-as-coverage, NOT for physically-based transparency (i.e. this extension). If alpha-as-coverage is not being used, the blend mode of the material should be set to "OPAQUE" even though it is transparent. Again, it's helpful to think of alpha-as-coverage as whether the physical surface is there or not. `transmission` applies to the surface material that exists.
```

Comment:
I did notice that in the sample model, all materials are flagged as "BLENDED".  While on the topic of the sample model, it requires the KHR_texture_transform extension; that dependency should be removed.

Reply:
Good catch. Only the ones with opacity should have had that. This is fixed now.

Reply:
Since the sample model has appeared in https://github.com/KhronosGroup/glTF-Sample-Models/pull/265, can the sample files associated with this PR be removed?  I'd like for there not to be two sources of sample models (although I realize some other extensions already violate this).

Reply:
Will do.